flink-cdc-connect/flink-cdc-source-connectors/flink-connector-debezium/src/main/java/org/apache/flink/cdc/debezium/table/DebeziumOptions.java

```diff
@@ -43,6 +47,10 @@ public static Properties getDebeziumProperties(Map<String, String> properties) {
             throw new IllegalArgumentException(
                     "table.include.list and table.exclude.list are not supported to set manually, please remove these options.");
         }
+        if (debeziumProperties.containsKey(COLUMN_INCLUDE_LIST.name())
+                && debeziumProperties.containsKey(COLUMN_EXCLUDE_LIST.name())) {
+            throw new IllegalArgumentException(TABLE_INCLUDE_LIST_ALREADY_SPECIFIED_ERROR_MSG);
+        }
```

Comment:
COLUMN_INCLUDE_LIST and COLUMN_EXCLUDE_LIST cannot coexist https://github.com/debezium/debezium/blob/ce9ddb119eaebef4a78882146f20a3d3ebd7c30f/debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java#L743